python/ray/train/torch/torch_trainer.py

```diff
@@ -45,32 +46,33 @@ def train_loop_per_worker(config: Dict):
     Inside the ``train_loop_per_worker`` function, you can use any of the
     :ref:`Ray AIR session methods <air-session-ref>`.
 
-    .. code-block:: python
+    .. testcode::
 
         def train_loop_per_worker():
             # Report intermediate results for callbacks or logging and
             # checkpoint data.
+            #
```

Comment:
I feel like it was better without this line but if you prefer feel free to keep it :)

Reply:
Since the code is incomplete `session.report(...)` and `session.get_checkpoint()`, nice to explain it with a comment